station/app/api/api_v1/endpoints/local_trains.py

```diff
@@ -271,7 +227,7 @@ async def get_file(train_id: str, file_name: str):
     return Response(file)
 
 
-@router.get("/{train_id}/getLogs")
+@router.get("/{train_id}/logs", response_model=list[lt_schemas.LocalTrainLog])
 def get_logs(train_id: str, db: Session = Depends(dependencies.get_db)):
```

Comment:
```python
def get_logs(train_id: str, all: bool = False, db: Session = Depends(dependencies.get_db)):
```
Use query parameter to control which logs to get i.e. if not all only return most recent

Reply:
done